### PR TITLE
Use MakerNegotiator.getUrl(), remove steps for `TakerNegotiator`

### LINKED ISSUE #371: Link to maker is confusing
Similar to https://github.com/comit-network/comit-js-sdk/issues/88

## Problem

https://github.com/comit-network/create-comit-app/blob/master/create/new_project/examples/separate_apps/src/maker.ts#L87

This _invitation link_ 
- exposes internal wire of the sdk (use of http for negotiation communications)
- is hard coded

Because of that, instead of being inspiring, it just creates more confusion.

## Goal

- Do not expose unnecessary details to developer
- Provide appropriate function to the developer via the SDK
- Update sdk, release and update example

## Recommendation

I see several choices:
1. ~Just remove it from example.~
1. Modify the SDK to provide a function that returns this value and be clearer on what it would be used for: `makerNegotiator.getUrl() // Returns the an url that can be used by a taker to trade with us.` (would need to move this issue to sdk repo)

_Edit: After writing #comit-js-sdk#90 I'd be more keen to go for 2._

### PR DESCRIPTION
**Pull request by D4nte**
Resolves #371 and comit-network/comit-js-sdk#90.

### CODE REVIEW

File: create/new_project/examples/btc_eth/src/maker.ts [hunk 1]
```diff
@@ -59,9 +59,8 @@ import { createActor, sleep } from "./lib";
     );
 
     // Start the HTTP service used to publish orders.
-    const makerHttpApi = new MakerHttpApi(makerNegotiator);
     // The maker's HTTP service will be served at port 2318.
```
Comment (luckysori) [inline create/new_project/examples/btc_eth/src/maker.ts]: 🙃 
```suggestion
     // The maker's HTTP service will be served at http://localhost:2318.
```

File: create/new_project/examples/btc_eth/src/maker.ts [hunk 2]
```diff
@@ -82,10 +81,10 @@ import { createActor, sleep } from "./lib";
     // Publish the order so the taker can take it.
     makerNegotiator.addOrder(order);
 
-    // Let the world know that there is an order available.
-    // In a real-world application this information could be shared publicly, e.g. on social medias.
-    const invitationDetails = `http://localhost:2318/orders/ETH-BTC`;
-    console.log(`Waiting for someone taking my order at: ${invitationDetails}`);
+    // Let the world know that you are a maker.
+    // Your app could publish this link on forum or social medias for takers to connect to.
```
Comment (luckysori) [inline create/new_project/examples/btc_eth/src/maker.ts]: 🔧 
```suggestion
    // Your app could publish this link on forum or social media for takers to connect to.
```

File: create/new_project/examples/btc_eth/src/taker.ts [hunk 3]
```diff
@@ -33,13 +33,19 @@ import { createActor, sleep } from "./lib";
     // Wait for commandline input for demo purposes
     readLineSync.question("1. Ready to accept and order from the maker?");
 
+    // This URL needs to be provided by the other party, the maker.
+    // `MakerNegotiator.getUrl()` on the Maker side returns this url
```
Comment (luckysori) [inline create/new_project/examples/btc_eth/src/taker.ts]: 🤔 
```suggestion
    // `MakerNegotiator.getUrl()` on the Maker side returns this url.
    // It's only hardcoded here for simplicity.
```

File: create/new_project/examples/btc_eth/src/maker.ts [hunk 4]
```diff
@@ -82,10 +81,10 @@ import { createActor, sleep } from "./lib";
     // Publish the order so the taker can take it.
     makerNegotiator.addOrder(order);
 
-    // Let the world know that there is an order available.
-    // In a real-world application this information could be shared publicly, e.g. on social medias.
-    const invitationDetails = `http://localhost:2318/orders/ETH-BTC`;
-    console.log(`Waiting for someone taking my order at: ${invitationDetails}`);
+    // Let the world know that you are a maker.
+    // Your app could publish this link on forum or social media for takers to connect to.
```
Comment (da-kami) [inline create/new_project/examples/btc_eth/src/maker.ts]:
```suggestion
    // Your app could publish this link on a forum or social media so takers can connect to you.
```

**Comment by D4nte** [comment]
bors retry

**Pull request merged**